### ISSUE TITLE
Update 'how to add a new volume' documentation

### ISSUE THREAD

**Issue by mialondon** [LibCrowds/libcrowds#850]
As per https://github.com/LibCrowds/libcrowds/issues/843#issuecomment-524083526

First: find the current documentation

**Comment by mialondon** [comment]
Notes from Geoffroy Noël copied over from #843

Here is how I managed to create a volume, a project and import the tasks:

    create a new volume in the admin by first selecting playbill collection then going to Volumes page and clicking the New button;
    fill in the form (Name, short name and importer = iiif) and submit
    go to the "Import Source" tab and insert the link to the manifest (please try that link in another browser tab first to make sure it actually returns a json file)
    click "update import source" button and wait for the success message
    Just to make sure, you can reload the page and go to the "Import Source" tab to verify that the value isn't empty (many problematic volumes do have an empty value)

6 . Create a new project in the usual way (/admin/project/new/playbills)
7. On the creation form select in the template (e.g. Dates), the newly created volume, ... and press Create
8. It should show a success message
9. After the screen reloads, tag the Task, Entity and Location then Click Update

After step 8, Libcrowds will create a background job of Medium priority to import the tasks from the manifest. This job is queued and can therefore take several minutes before it comes at the top. You should be able to see that job in the Admin:

    select to "Background Tasks" on the left-hand side panel
    look at the Queue table and check if you have a Medium Job,
    it should say 1 (or more) if the import job was created, 0 if the import is already finished

You can also see those import tasks on your admin Dashboard, under the New Task section.

**Comment by mialondon** [comment]
Some additional documentation and previous discussion that will be useful in understanding the In the Spotlight volume loading issue:
 
https://github.com/LibCrowds/libcrowds/issues/862 - the current issue. As mentioned in the discussion, we've noted actions taken at specific times, which can be linked to the extensive collection of trace errors
 
https://github.com/LibCrowds/libcrowds/issues/843 - an original report of a loading bug, with some work by an external agency
 
https://github.com/LibCrowds/libcrowds/issues/850 - steps to add a new volume to the backend, which should provide opportunities to see the bug in action
 
https://docs.libcrowds.com/ - documentation for the LibCrowds platform. If there are areas where this is incomplete, we can look to update it as relevant information is uncovered.
 
https://docs.pybossa.com/ - documentation for the underlying Pybossa platform
 
https://vue-a11y.com/ - accessibility fixes from the vue.js community that might help address other issues
